examples/ale_regression_boston.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<ul><li>We can separate _in this case_ in between commas as it's an incident construction?</li><li>I think we should refer to the present plot given one is plotted so we can say "The ALE on the y-axis of the plot above ..." or even "The units of the y-axis of the plot above" ... since it's clear from the label we are plotting the ALE value...</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>